pokete_classes/npcs.py

```diff
@@ -203,7 +202,7 @@ class Trainer(NPC, Provider):
     def __init__(self, pokes, name, gender, texts, lose_texts,
                  win_texts):
         NPC.__init__(self, name, texts, side_trigger=False)
-        Provider.__init__(self, pokes)
+        Provider.__init__(self, pokes, False, 2)
```

Comment:
might be nice to use keyword args here so it's easy to see what `False` and `2` are referring to.